src/Views/Format.elm

```diff
@@ -104,6 +106,30 @@ formatRichFloat decimals unit value =
         ]
 
 
+complement : Unit.Impact -> Html msg
+complement impact =
+    -- Notes:
+    -- - maluses are expressed with a negative number, bonuses with a
+    --   positive one; here we render the *effect* it has on the score
+    -- - complements are *always* expressed in ecoscore points
```

Comment:
`s/ecoscore/environmental cost`?

Also, I know it kind of makes sense to have maluses as negative numbers, and their effect being positive, but it's really confusing. Not sure if there's a better way to do it though... maybe a union type `type Complement = Bonus Quantity | Malus Quantity`?

Reply:
I agree and suggest we address that in another PR as this would involve updating too many unrelated places to the scope of the current issue addressed by this patch.